e2e/regression_settings/settings.no_max_basal.spec.js

```diff
@@ -0,0 +1,17 @@
+const { loop } = require('../../src/index');
+
+describe('Closed loop is not allowed when settings are not applied for max basal', () => {
+    it('launch loop', async () => {
+        await loop.Launch();
+    });
+    it('setup without max basal applied', async () => {
+        await loop.screens.settings.Open();
+        await loop.screens.settings.AddPumpSimulator();
+        var checks = async function () {
```

Comment:
Is this so you can do these checks before closing the specific settings window? If so, then this makes sense, but I might call them `expectations` rather than `checks` (since Detox uses "expect" terminology).